Makefile

```diff
@@ -30,8 +32,10 @@ endif
 all: image
 
 image:
-	docker build --secret id=SLES_REGISTRATION_CODE --pull ${DOCKER_ARGS} --tag '${NAME}:${VERSION}' .
-	docker tag '${NAME}:${VERSION}' ${NAME}:${VERSION}-${TIMESTAMP}
-	docker tag '${NAME}:${VERSION}' ${NAME}:${SLES_VERSION}
-	docker tag '${NAME}:${VERSION}' ${NAME}:${SLES_VERSION}-${VERSION}
-	docker tag '${NAME}:${VERSION}' ${NAME}:${SLES_VERSION}-${VERSION}-${TIMESTAMP}
+	docker buildx create --use
+	docker buildx build --platform=linux/amd64,linux/arm64 --secret id=SLES_REGISTRATION_CODE --pull ${DOCKER_ARGS} .
```

Comment:
I'm not familiar with the `buildx build` action here, is this adding both platforms in parallel or is it building one after the other (sequentially)?

Reply:
It builds both ARCH's simultaneously.